namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-ui-staging/resources/route53.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_route53_zone" "lcdui_route53_zone" {
```

Comment:
Hi @kmahern 
It is better to maintain this route53 zone in your Prod namespace.
If you don't have prod namespace yet, can you create one and have this zone creation in that namespace please.

Reply:
👍 Hi @vijay-veeranki-moj, I have created the prod namespace and moved the route53 zone maintenance into it.